modules/oxide/src/less/theme/components/notification/notification.less

```diff
@@ -7,27 +7,43 @@
 @notification-font-size: @font-size-sm;
 @notification-font-weight: @font-weight-normal;
 @notification-padding: @pad-xs;
+@notification-link-text-decoration: underline;
+@notification-link-hover-text-decoration: underline;
+@notification-link-active-text-decoration: underline;
+@notification-link-focus-outline-border-radius: 1px;
 
 @notification-success-background-color: mix(@background-color, @color-success, 80%);
 @notification-success-border-color: mix(@background-color, @color-success, 70%);
 @notification-success-text-color: @text-color;
 @notification-success-link-color: mix(@text-color, @color-success, 45%);
+@notification-success-link-hover-color: darken(@notification-success-link-color, 20%);
+@notification-success-link-active-color: darken(@notification-success-link-color, 30%);
+@notification-success-link-focus-outline-color: @notification-success-link-color;
```

Comment:
Does this match the link outline from the help dialog?

Reply:
The background and text colour uses a different colour compared to the help dialog background, and the styles for link outline from the help dialog fail on colour contrast 😢